packages/flash/src/index.ts

```diff
@@ -1,5 +1,5 @@
 import { AsyncLocalStorage } from 'node:async_hooks';
-import { Request, Response, NextFunction } from 'express';
+import type { Request, Response, NextFunction } from 'express';
```

Comment:
Importing as type means this can be a dev dependency. As it was it required a regular dependency.

Reply:
That's not true; does dependency-cruiser think it is? Even if we "only" use types, in this case they very much end up in `dist/index.d.ts`, which in turn means they're needed for consumers of this package. (I know I originally authored this `package.json` 🙃 this was my mistake in the first place!)

Reply:
Ok, makes sense. Anyway, the rules for dependency-cruiser in this case are here:
https://github.com/PrairieLearn/PrairieLearn/blob/003032f201c14a7232bc07f1b136676480f2d90f/.dependency-cruiser.js#L78

In essence, if the import is type-only, then the `not-to-dev-dep` rule does not apply. If you have a suggestion to refine this further that might catch cases like this. In any case, I can add the dependency as a regular one in the meantime (or I can hold off if you want a scenario to test with).